README.md

```diff
@@ -1,24 +1,227 @@
+  <h3><a href="https://github.com/Amtresu/Rent-A-Car/">Link to the FRONT-END Repo</a></h3>
+  <hr>
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Book an Appointment for Cars<a name="about-project"></a>
 
-* Database creation
+**Book an Appointment for Cars** is a backend application used for creating API for [Rent-A-Car](https://github.com/Amtresu/Rent-A-Car/) Web application.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- create different API end points those are required for Rent-a-Car web app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Loom Video Link - not applicable yet](https://www.loom.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+````
+git clone https://github.com/PRATAP-KUMAR/book-an-appointment
+cd book-an-appointment
+ls
+````
+
+### Prerequisites
+
+In order to run this project you need:
+
+- a browser
+- a terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+````
+gem install rails
+````
+
+### Usage
+
+To run the project, execute the following command from the root directory:
+
+```sh
+  rails server
+```
+then open any browser and enter this address `localhost:3000`
+
+### Run tests
+````sh
+rspec
+````
+
```

Comment:
- [ ] Your tests work, however, the instructions on your readme state that they should be run only using the `rspec` command.  This results in failed tests:

![image](https://user-images.githubusercontent.com/57421823/212439633-11d375b3-7e96-4a75-b3ea-43cc06dc670e.png)

Soilution:

Modify your readme instructions so the commands are specific to each file:

```bash
rspec spec/requests/cars_spec.rb
rspec spec/requests/reservations_spec.rb
rspec spec/requests/users_spec.rb

rspec spec/models/car_spec.rb
rspec spec/models/user_spec.rb

rake rswag
```

That way, it will be more specific, and all the tests will pass. 💪

Reply:
Will fix this. Thanks.